zsh-aws-vault.plugin.zsh

```diff
@@ -68,6 +68,12 @@ function avli() {
       com.google.chrome)
         echo "${login_url}" | xargs -t nohup /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome %U $AWS_VAULT_PL_BROWSER_LAUNCH_OPTS --no-first-run --new-window --disk-cache-dir=$(mktemp -d /tmp/chrome.XXXXXX) --user-data-dir=$(mktemp -d /tmp/chrome.XXXXXX) > /dev/null 2>&1 &
         ;;
+      com.microsoft.edgemac)
+        echo "${login_url}" | xargs -t nohup /Applications/Microsoft\ Edge.app/Contents/MacOS/Microsoft\ Edge %U $AWS_VAULT_PL_BROWSER_LAUNCH_OPTS --inprivate --no-first-run --new-window --disk-cache-dir=$(mktemp -d /tmp/msedge.XXXXXX) --user-data-dir=$(mktemp -d /tmp/msedge.XXXXXX) > /dev/null 2>&1 &
```

Comment:
Should we use `--inprivate` here? With the other flags we're providing, a new instance of Edge will open each time `avli` is run. So I wonder if `--inprivate` is really giving us any added functionality.

My gut is that we should match the other chromium-like flags (e.g., remove `--inprivate`) and allow people to pass this using `$AWS_VAULT_PL_BROWSER_LAUNCH_OPTS` if they want this behavior.

Is there some other behavior `--inprivate` gives users that I'm not aware of?

Reply:
you are correct, have tested and verified behaviour is as expected without the flag